ddocs/medic/_attachments/translations/messages-en.properties

```diff
@@ -1295,3 +1295,5 @@ permission.description.can_view_tasks_tab = Display the Tasks tab in the app. If
 permission.description.can_view_unallocated_data_records = Allowed to see reports that have no assigned contact.
 permission.description.can_view_users = Allowed to get a list of all configured users.
 permission.description.can_write_wealth_quintiles = Allowed to update contacts with their wealth quintiles.
+Display = Display
+date.time.title = Date & time 
```

Comment:
The new translate key style is to have all lowercase. You should also have more context so this key is only used where appropriate - "Display" is too generic. `admin.display` and `admin.display.datetime` would be better.